pages/features/trail.mdx

```diff
@@ -2,4 +2,12 @@ import { Callout } from 'nextra-theme-docs'
 import { Card, Cards } from 'nextra-theme-docs'
 import Image from 'next/image'
 
-# Trail 
\ No newline at end of file
+# Trail
+
+Trails are a key feature on Learnhouse that enables teachers & students to track their progress through Courses.
+
+By using Trails, teachers can effectively use quiz results and text-based course progression to tailor feedback in-person & remotely. It also lets students keep a track of their own progression through the course, and encourage increased self-determination & confidence.
+
+By using this feature, teachers or course content creators can collaboratively view and track student progress to address weak spots in content, to aid with revision, and for consolidation further down in the course.
```

Comment:
This is a little bit further than what the feature really is, as for now it serves the only purpose of showing users their progression through a given course
```suggestion

```